.gitignore

```diff
@@ -1,2 +1,3 @@
 /.idea/
 .DS_Store
+/site/.vitepress
```

Comment:
Seriously?

Reply:
What is the point of that?

Reply:
what? why? I am asking the same question here. lol

Reply:
Let me fix the merge conflict first

Reply:
Oh. Turns out it was due to the default cache directory location being there. I have moved the location to `./site/docs/.vitepress` (notice the `docs`) -> https://github.com/grammyjs/website/pull/824/commits/5825a097d9a2a64973f7580e9d2be07bd26beafb